src/components/mgt-login/mgt-login.ts

```diff
@@ -33,6 +33,18 @@ export class MgtLogin extends MgtBaseComponent {
   })
   userDetails: MgtPersonDetails;
 
+  @property({
+    attribute: 'sign-in-text',
```

Comment:
Docs needed

Reply:
I'll remove these for now, originally I had them in the templating stuff below, but had removed only part.

I created an issue #102 about localization, as we should figure out what general strategy we want to use across controls.